### PR TITLE
BMS-759: Update URLs in the PlayNetwork SDK

### PR DESCRIPTION
**Pull request by jcarva**
[BMS-759 Ticket](https://jira.touchtunes.com/browse/BMS-759)

- Create host constant file
- Update URLs

### CODE REVIEW

File: lib/hosts.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module.exports = {
+    ASSET_API: 'master-assetapi.scaffold-workers-ext-main-us.m.cld.octavelive.com/v1/version',
```
Comment (gcerrott) [inline lib/hosts.js]: When comparing before and after, it looks like some constants in this file have a path appended which was not part of the literal in the original file.  Could you explain?

**Review by gcerrott — CHANGES_REQUESTED** [review]
Also,
 - we might want to bump up the version.
 - I don't think the PR is targeting the right branch

File: lib/hosts.js (hunk 1 above)
Comment (jcarva) [reply lib/hosts.js]: I got all the urls from https://wiki.touchtunes.com/display/PIC/api.apps.playnetwork.com#api.apps.playnetwork.com-Plan, in the **_Current List_** section.

I'll remove the paths and also leave a comment on the wiki page.

**Review by jcarva — COMMENTED** [review]
I'll update the version and the target branch.

**Pull request closed without merging**